lua/lualine/themes/onedark.lua

```diff
@@ -6,6 +6,7 @@ local colors = {
   blue   = '#61afef',
   green  = '#98c379',
   purple = '#c678dd',
+  cyaan  = '#56b6c2',
```

Comment:
I referred to this color code
https://github.com/navarasu/onedark.nvim/blob/master/lua/onedark/colors.lua#L16